components/sock_utils/src/gai_strerror.c

```diff
@@ -1,17 +1,36 @@
 /*
- * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ * SPDX-FileCopyrightText: 2024-2025 Espressif Systems (Shanghai) CO LTD
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 #include <stdio.h>
 #include "gai_strerror.h"
+#include "lwip/netdb.h"
 
-_Thread_local char gai_strerror_string[32];
+#define HANDLE_GAI_ERROR(code) \
+    case code: return #code;
 
-const char *gai_strerror(int ecode)
+const char *gai_strerror(int errcode)
```

Comment:
Function definition is still `const char *gai_strerror(int ecode);` in header file.

Reply:
Thanks for noticing the inconsistency. Fixed the prototype in headers.